lte/gateway/c/core/oai/tasks/nas5g/src/ies/M5GUESecurityCapability.cpp

```diff
@@ -98,7 +98,15 @@ int UESecurityCapabilityMsg::DecodeUESecurityCapabilityMsg(
     ue_sec_capability->eia7 = *(buffer + decoded) & 0x1;
     decoded++;
 
-    // decoded = type_len + length_len + ue_sec_capability->length;
+    // Skipping the remaining bytes as not supported
+    if (ue_sec_capability->length > (decoded - (type_len + length_len))) {
+      // The length of the bytes skipped
+      uint8_t add_ue_sec_cap_length =
+          ue_sec_capability->length - (decoded - (type_len + length_len));
```

Comment:
Line numbers 105 and 107 are same i.e ue_sec_capability->length - (decoded - (type_len + length_len)). 
decoded += add_ue_sec_cap_length;
ue_sec_capability->length -= add_ue_sec_cap_length;

Reply:
Addressed Review comment